HLTrigger/Configuration/python/HLTrigger_EventContent_cff.py

```diff
@@ -400,7 +400,15 @@
         'keep *_hltScoutingPFPacker_*_*',
         'keep *_hltScoutingPrimaryVertexPacker_*_*',
         'keep *_hltScoutingTrackPacker_*_*',
-        'keep edmTriggerResults_*_*_*'
+        'keep edmTriggerResults_*_*_*',
+        'keep *_hltL2MuonSeedsFromL1TkMuon_*_*',
+        'keep *_hltL2MuonsFromL1TkMuon_*_*',
+        'keep *_hltIter2Phase2L3FromL1TkMuonMerged_*_*',
+        'keep *_hltPhase2L3OIMuonTrackSelectionHighPurity_*_*',
+        'keep *_hltPhase2L3MuonMerged_*_*',
+        'keep *_hltPhase2L3GlbMuon_*_*',
+        'keep *_hltPhase2L3MuonsNoID_*_*',
+        'keep *_hltPhase2L3Muons_*_*'
```

Comment:
This is an auto-generated file for Run3, any changes will be lost! Please remove the change to this file.

Instead, you'd need to add your changes, using an era/modifier approach, here:
https://github.com/cms-sw/cmssw/blob/master/Configuration/EventContent/python/EventContent_cff.py
see there for examples (eg, Phase2 tracker etc.).

Reply:
@Martin-Grunewald already commented at https://github.com/cms-sw/cmssw/pull/46860#discussion_r1869060089